eng/pipelines/templates/steps/azd-login.yml

```
@@ -0,0 +1,20 @@
+parameters:
+  SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+  AzdDirectory: "cli/azd"
+
+steps:
+  - pwsh: |
+      $subscriptionConfiguration = @'
+        ${{ parameters.SubscriptionConfiguration }}
+      '@ | ConvertFrom-Json -AsHashtable;
+
+      ./azd login `
```

Probably want `$LASTEXITCODE` checks to avoid silent errors